src/Sulu/Bundle/AdminBundle/FormMetadata/FormMetadataMapper.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\FormMetadata;
+
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FieldMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\FormMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\OptionMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\SectionMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\FormMetadata\TagMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata\PropertyMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata\SchemaMetadata;
+use Sulu\Component\Content\Metadata\BlockMetadata;
+use Sulu\Component\Content\Metadata\ItemMetadata as ContentItemMetadata;
+use Sulu\Component\Content\Metadata\PropertyMetadata as ContentPropertyMetadata;
+use Sulu\Component\Content\Metadata\SectionMetadata as ContentSectionMetadata;
+
+/**
+ * Maps the deprecated form metadata objects to the actual ones.
+ */
+class FormMetadataMapper
+{
+    /**
+     * @param array $children
+     * @param FormMetadata $form
+     * @param string $locale
+     *
+     * @throws \Exception
+     */
+    public function mapChildren(array $children, FormMetadata $form, string $locale)
+    {
+        foreach ($children as $child) {
+            if ($child instanceof BlockMetadata) {
+                $item = $this->mapBlock($child, $locale);
+            } elseif ($child instanceof ContentPropertyMetadata) {
+                $item = $this->mapProperty($child, $locale);
+            } elseif ($child instanceof ContentSectionMetadata) {
+                $item = $this->mapSection($child, $locale);
+            } else {
+                throw new \Exception('Unsupported property given "' . get_class($child) . '"');
+            }
+
+            $form->addItem($item);
+        }
+    }
+
+    /**
+     * @param array $itemsMetadata
+     *
+     * @return SchemaMetadata
```

Comment:
We started to avoid these PHPDoc comments, when they don't add any additional value (These don't, because all the information is also in the types of the method signature.